developers/weaviate/current/graphql-references/get.md

```diff
@@ -76,14 +76,14 @@ To combine `Get { }` with a vector search argument, here is an overview of the s
 
 | Argument | Description | Required Modules (at least one of) | Learn More |
 | --- | --- | --- | --- |
-| `nearObject` | Find the nearest neighbors of an object referenced by its id | *none - works out of the box* | [Learn more](../graphql-references/filters.html#nearobject-vector-search-argument) |
-| `nearVector` | Find the nearest neighbors to any vector | *none - works out of the box* | [Learn more](../graphql-references/filters.html#nearvector-vector-search-argument) |
-| `nearText` | Vectorize a text query and perform a vector search based on it | `text2vec-transformers`, `text2vec-contextionary`, `text2vec-openai`, `multi2vec-clip`, `text2vec-huggingface` | [Transformers](../retriever-vectorizer-modules/text2vec-transformers.html#how-to-use), [Contextionary](../retriever-vectorizer-modules/text2vec-contextionary.html#how-to-use), [OpenAI](../retriever-vectorizer-modules/text2vec-openai.html#how-to-use), [CLIP](../retriever-vectorizer-modules/multi2vec-clip.html#how-to-use), [Huggingface](../retriever-vectorizer-modules/text2vec-huggingface.html#how-to-use) |
+| `nearObject` | Find the nearest neighbors of an object referenced by its id | *none - works out of the box* | [Learn more](vector-search-parameters.html#nearobject) |
+| `nearVector` | Find the nearest neighbors to any vector | *none - works out of the box* | [Learn more](vector-search-parameters.html#nearvector) |
+| `nearText` | Vectorize a text query and perform a vector search based on it | `text2vec-transformers`, `text2vec-contextionary`, `text2vec-openai`, `multi2vec-clip`, `text2vec-huggingface` | [Transformers](../retriever-vectorizer-modules/text2vec-transformers.html#how-to-use), [Contextionary](../retriever-vectorizer-modules/text2vec-contextionary.html#how-to-use), [OpenAI](../retriever-vectorizer-modules/text2vec-openai.html#how-to-use), [CLIP](../retriever-vectorizer-modules/multi2vec-clip.html#how-to-use), [Huggingface](../retriever-vectorizer-modules/text2vec-huggingface.html#how-to-use), [Cohere](../retriever-vectorizer-modules/text2vec-cohere.html#how-to-use) |
 | `nearImage` | Vectorize an image and perform a vector search based on it | `multi2vec-clip`, `img2vec-neural` | [CLIP](../retriever-vectorizer-modules/multi2vec-clip.html#neartext), [Img2Vec](../retriever-vectorizer-modules/img2vec-neural.html#nearimage-search) |
 
 # Filters
 
-`Get{}` functions can be extended with search filters (both semantic filters as traditional filters). Because the filters work on multiple core functions (like `Aggregate{}`) there is a [specific documentation page dedicated to filters](filters.html).
+`Get{}` functions can be extended with search filters (both semantic filters and traditional filters). Because the filters work on multiple core functions (like `Aggregate{}`) there is a [specific documentation page dedicated to filters](filters.html).
```

Comment:
What exactly are "semantic filters"? Is this another name for [vector search parameters](https://weaviate.io/developers/weaviate/current/graphql-references/vector-search-parameters.html)? That page starts by referring to concepts like `nearVector` by the term "vector search parameter" but continues with "filters". Further, the [`img2vec` page](https://weaviate.io/developers/weaviate/current/retriever-vectorizer-modules/img2vec-neural.html#nearimage-search) calls `nearVector` an "operator". 😕 If we're talking about the same concept, we should use choose terminology.

CC @sebawita @databyjp

Reply:
Yup. I agree and propose a separate review for terminology and glossary over the next little while. (Bandwidth permitting.) I will create an issue actually.